### ISSUE TITLE
UI Idea: Buff Timers on Screen

### ISSUE THREAD

**Issue by novemr** [Daenks/Meridian59_103#994]
I'd really like to see something like that, it makes building/fighting way more relaxing...

![image](https://cloud.githubusercontent.com/assets/12058341/7552452/1dd73750-f6b5-11e4-8815-7915eb29306f.png)

**Comment by jfaubert087** [comment]
Cool idea.

**Comment by treymd** [comment]
Yep, it's a good idea, research will need to be done on how difficult that would be..

**Comment by tsShergill** [comment]
This would be a really cool feature to have!
